src/test/java/roomescape/admin/controller/AdminControllerTest.java

```diff
@@ -0,0 +1,174 @@
+        }
+
+        @Test
+        @DisplayName("목록을 조회할 수 있습니다.")
+        @Rollback
+        void return_list() throws JsonProcessingException {
+            save_reservation();
+
+            var response = RestAssured
+                    .given().log().all()
+                    .contentType(ContentType.JSON)
+                    .when().get("/reservations")
+                    .then().log().all().extract();
+
+            // 응답을 String으로 추출합니다.
+            String jsonResponse = response.asString();
+
+            // ObjectMapper를 사용하여 JSON 응답을 List로 변환합니다.
+            ObjectMapper objectMapper = new ObjectMapper();
+            List<Object> reservations = objectMapper.readValue(jsonResponse, new TypeReference<List<Object>>() {});
+
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            assertThat(reservations.size()).isGreaterThan(0);
+        }
+
+        @Nested
+        @DisplayName("잘못 등록되었다면")
+        class Context_Wrong_Resevation{
+
+            @Test
+            @DisplayName("취소하실 수 있습니다.")
+            @Rollback
+            void delete_reservation(){
+                save_reservation();
+
+                var response = RestAssured
+                        .given().log().all()
+                        .contentType(ContentType.JSON)
+                        .when().delete("/reservations/1")
+                        .then().log().all().extract();
+
+                assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("예약 시간은")
+    class Describe_Reservation_Time{
+
+        void save_reservation_time(SaveReservationTimeRequest saveReservationTimeRequest){
+            RestAssured.given().log().all()
+                    .body(saveReservationTimeRequest)
+                    .contentType(ContentType.JSON)
+                    .when().post("/times")
+                    .then().log().all().extract();
+        }
+
+        @Test
+        @DisplayName("목록을 조회할 수 있습니다.")
+        @Rollback
+        void return_list() throws JsonProcessingException {
+            save_reservation_time(given_예약시간_10_20);
+            save_reservation_time(given_예약시간_13_20);
+
+            var response = RestAssured
+                    .given().log().all()
+                    .contentType(ContentType.JSON)
+                    .when().get("/times")
+                    .then().log().all().extract();
+
+            // 응답을 String으로 추출합니다.
+            String jsonResponse = response.asString();
+
+            // ObjectMapper를 사용하여 JSON 응답을 List로 변환합니다.
+            ObjectMapper objectMapper = new ObjectMapper();
+            List<Object> reservations = objectMapper.readValue(jsonResponse, new TypeReference<List<Object>>() {});
+
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+            assertThat(reservations.size()).isEqualTo(2);
```

Comment:
`hasSize()` 를 사용할 수 있을 것 같아요 ~ 

```suggestion
            assertThat(reservations).hasSize(2);
```

Reply:
isEqualTo를 사용하지 않고 hasSize를 사용하는게 더 명확하고 간단하네요..!
참고하겠습니다ㅎ